### DISCUSSION TITLE
Trouble getting nextjs `npm run dev` to work properly

### DISCUSSION THREAD

**Discussion by eliluong** [coder/code-server#6735] [Q&A]
Hello, I am having some issues getting NextJS to work properly.

I am using sshfs and a volume mount for my data: `"/home/eliluong/code-server/code:/home/coder/code"`

I have created a project with `npx create-next-app@latest` and `npm run dev -- --port 3100` to use port 3100. I have code-server run on a domain, `code.mydomain.com`, and so it ends up getting accessed via `https://code.mydomain.com/proxy/3100/`. But everything is broken.

![image](https://github.com/coder/code-server/assets/15806938/8d935db0-6595-4952-9b8a-9c176e979b1f)

If I go to the server that contains the data that I mounted via sshfs, and do `npm run dev` there, and access it directly then it loads properly.

I feel this is a common problem, but I wasn't able to find a thread. The browser console is full of 404 errors, so I think it's some path-related issue. Thank you.

**Comment by code-asher** [comment]
Yeah there have been a few threads (https://github.com/coder/code-server/discussions?discussions_q=is%3Aopen+nextjs) but I am not sure anyone actually got it working.  I think NextJS has a bug with a base path.  If you can use a subdomain proxy, that will be more reliable.
